File: ncDados/dClienteEndereco.vb
Namespace nsCliente

  Public Class ColecaoClienteEndereco
    Inherits List(Of dClienteEndereco)
  End Class

  Public Class dClienteEndereco

    Private _logradouro As String
    Private _numero As Nullable(Of Integer)
    Private _complemento As String
    Private _cidade As String
    Private _estado_cid As Nullable(Of Integer)
    Private _cep As Nullable(Of Integer)
    Private _dataInclusao As String
    Private _tipoResidencia As String
    Private _bairro As String
    Private _tipoEndereco As String
    Private _cliente_cid As Nullable(Of Integer)
    Private _valor As Nullable(Of Decimal)
        Private _tempo As String
        Private _siglaEstado As String

    Public Property logradouro() As String
      Get
        Return _logradouro
      End Get
      Set(ByVal value As String)
        _logradouro = value
      End Set
    End Property

    Public Property numero() As Nullable(Of Integer)
      Get
        Return _numero
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _numero = value
      End Set
    End Property

    Public Property complemento() As String
      Get
        Return _complemento
      End Get
      Set(ByVal value As String)
        _complemento = value
      End Set
    End Property

    Public Property cidade() As String
      Get
        Return _cidade
      End Get
      Set(ByVal value As String)
        _cidade = value
      End Set
    End Property

    Public Property estado_cid() As Nullable(Of Integer)
      Get
        Return _estado_cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _estado_cid = value
      End Set
    End Property

    Public Property cep() As Nullable(Of Integer)
      Get
        Return _cep
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _cep = value
      End Set
    End Property

    Public Property dataInclusao() As String
      Get
        Return _dataInclusao
      End Get
      Set(ByVal value As String)
        _dataInclusao = value
      End Set
    End Property

    Public Property tipoResidencia() As String
      Get
        Return _tipoResidencia
      End Get
      Set(ByVal value As String)
        _tipoResidencia = value
      End Set
    End Property

    Public Property bairro() As String
      Get
        Return _bairro
      End Get
      Set(ByVal value As String)
        _bairro = value
      End Set
    End Property

    Public Property tipoEndereco() As String
      Get
        Return _tipoEndereco
      End Get
      Set(ByVal value As String)
        _tipoEndereco = value
      End Set
    End Property

    Public Property cliente_cid() As Nullable(Of Integer)
      Get
        Return _cliente_cid
      End Get
      Set(ByVal value As Nullable(Of Integer))
        _cliente_cid = value
      End Set
    End Property

    Public Property valor() As Nullable(Of Decimal)
      Get
        Return _valor
      End Get
      Set(ByVal value As Nullable(Of Decimal))
        _valor = value
      End Set
    End Property

    Public Property tempo() As String
      Get
        Return _tempo
      End Get
      Set(ByVal value As String)
        _tempo = value
      End Set
        End Property
        Public Property siglaEstado() As String
            Get
                Return _siglaEstado
            End Get
            Set(ByVal value As String)
                _siglaEstado = value
            End Set
        End Property

  End Class

End Namespace

File: ncModelos/dClienteEndereco.cs
using System;
using System.Collections.Generic;

namespace nsCliente
{
    public class ColecaoClienteEndereco : List<dClienteEndereco>
    {
    }

    public class dClienteEndereco
    {
        private string _logradouro;
        private int? _numero;
        private string _complemento;
        private string _cidade;
        private int? _estado_cid;
        private int? _cep;
        private string _dataInclusao;
        private string _tipoResidencia;
        private string _bairro;
        private string _tipoEndereco;
        private int? _cliente_cid;
        private decimal? _valor;
        private string _tempo;
        private string _siglaEstado;

        public string logradouro
        {
            get => _logradouro;
            set => _logradouro = value;
        }

        public int? numero
        {
            get => _numero;
            set => _numero = value;
        }

        public string complemento
        {
            get => _complemento;
            set => _complemento = value;
        }

        public string cidade
        {
            get => _cidade;
            set => _cidade = value;
        }

        public int? estado_cid
        {
            get => _estado_cid;
            set => _estado_cid = value;
        }

        public int? cep
        {
            get => _cep;
            set => _cep = value;
        }

        public string dataInclusao
        {
            get => _dataInclusao;
            set => _dataInclusao = value;
        }

        public string tipoResidencia
        {
            get => _tipoResidencia;
            set => _tipoResidencia = value;
        }

        public string bairro
        {
            get => _bairro;
            set => _bairro = value;
        }

        public string tipoEndereco
        {
            get => _tipoEndereco;
            set => _tipoEndereco = value;
        }

        public int? cliente_cid
        {
            get => _cliente_cid;
            set => _cliente_cid = value;
        }

        public decimal? valor
        {
            get => _valor;
            set => _valor = value;
        }

        public string tempo
        {
            get => _tempo;
            set => _tempo = value;
        }

        public string siglaEstado
        {
            get => _siglaEstado;
            set => _siglaEstado = value;
        }
    }
}
